.github/actions/step_publish_test_results/action.yml

```diff
@@ -19,9 +19,6 @@ inputs:
     required: false
```

Comment:
`create_promptflow_release_tag` also have some references on `GITHUB_TOKEN`, may take a look

Reply:
permission is already provided in that file.